### DISCUSSION TITLE
播放列表中如果是mp3文件, 无法上一个和下一个!

### DISCUSSION THREAD

**Discussion by aeqvkec** [hooke007/mpv_PlayKit#467] [mpv 通用]
如题, 播放列表中如果是mp3文件, 无法上一个和下一个, 上一个和下一个按钮没有反应
PGUP 	playlist-prev				#播放列表上一个
PGDWN 	playlist-next			#播放列表下一个
按快捷键也无法播放上一个或下一个, 求助!

**Comment by hooke007** [comment]
playlist-prev 只对播放列表有用，你根本没在使用播放列表（先读一下最基本的说明，我假设你用的是懒人包）

**Reply by hooke007** [reply]
另外提问最好讲清楚你的使用情况。

我甚至不知道你在使用的是什么
我假设你在自己用了我的默认版本的uosc

https://github.com/hooke007/MPV_lazy/blob/20acfc0fd93202606916289fa422a7bc405ba0c4/portable_config/script-opts.conf#L472-L474

**Reply by aeqvkec** [reply]
不好意思啊, 没学过编程, 上网搜了很多使用说明, 有些还是没看明白. 
我使用的是懒人包, 上面的script-opts-append = uosc-autoload_types=video,audio 改好了, 上一个和下一个按钮正常了.

看了下播放列表, 只有一个文件, 如何把当前文件目录里的所有音视频文件自动添加到播放列表中? 谢谢!

**Reply by hooke007** [reply]
这里边不涉及编程

**Reply by hooke007** [reply]
自动添加到播放列表需要外部第三方脚本

**Reply by zhongfly** [reply]
https://github.com/mpv-player/mpv/blob/master/TOOLS/lua/autoload.lua

**Reply by aeqvkec** [reply]
好用的, 感谢! @zhongfly @hooke007

**Reply by aeqvkec** [reply]
您好, 查看了  uosc 多功能控制界面   还是不知道该如何写代码,   如何用快捷键来实现  上一个和下一个按钮   的功能?  (不是要操作播放列表)  求助!

**Reply by hooke007** [reply]
input里就有你要写啥
https://github.com/hooke007/MPV_lazy/blob/9713164e5bd2342c55c15902c690fd8095393aa0/portable_config/input_uosc.conf#L128-L129

**Reply by aeqvkec** [reply]
哦 no, 错过了这么重要的信息, 太感谢了, 这样mpv就完全符合我的预想了, 可以完美替代potplayer了, 感谢感谢!
